sdk/test-utils/recorder-new/src/utils/utils.ts

```diff
@@ -97,7 +97,7 @@ export interface RegexSanitizer {
   /**
    * A regex. Can be defined as a simple regex replace OR if groupForReplace is set, a subsitution operation.
    */
-  regex: string;
+  regex?: string;
```

Comment:
Is it optional because it is not used in playback mode? if `regex` is optional, is `value` optional too?

Reply:
This is only meant for convenience. If regex is undefined, the sanitizer won't be added.

For `value`, typically it is hard-coded and is a non-null value. 
If it turned out to be undefined, it is most likely that the user is doing something wrong/unexpected, hence I didn't think of allowing that.